lib/mapbox/geocoder.rb

```diff
@@ -6,17 +6,27 @@ class Geocoder
     include Mapbox::APIOperations::Request
     extend Mapbox::HashUtils
 
-    def self.geocode_forward(query, proximity=nil, dataset='mapbox.places')
-      proximity_param = ''
+    def self.geocode_forward(query, options={}, dataset='mapbox.places')
+      proximity = options[:proximity] || nil
```

Comment:
since the default value of an unset property is `nil`, are these ||s only to save people who put false in their place?